tsdb/engine/tsm1/string_test.go

```diff
@@ -63,7 +63,7 @@ func Test_StringEncoder_Multi_Compressed(t *testing.T) {
 		t.Fatalf("unexpected encoding: got %v, exp %v", b[0], stringCompressedSnappy)
 	}
 
-	if exp := 47; len(b) != exp {
+	if exp := 51; len(b) != exp {
```

Comment:
I looked through the diff of what changed in snappy (https://github.com/golang/snappy/compare/5979233c5d6225d4a8e438cdd0b411888449ddab...d9eb7a3d35ec988b8585d4a0068e462c27d28380) a bit, but it wasn't obvious to me why the encoded length went up by 4 bytes - presumably an implementation detail, maybe it will change again in the future? 

Would it be better to just assert that the encoded length is less than some fraction of the input length?